uint32[620] sensor1
uint32[620] sensor2
uint32[620] sensor3
uint32[620] sensor4
